webknossos/webknossos/client/_upload_dataset.py

```diff
@@ -67,7 +68,7 @@ def upload_dataset(dataset: Dataset) -> str:
     with Progress() as progress:
         with Resumable(
             f"{datastore_url}/data/datasets?token={datastore_token}",
-            simultaneous_uploads=5,
+            simultaneous_uploads=1 if "PYTEST_CURRENT_TEST" in os.environ else 5,
```

Comment:
@normanrz The parallelism here was a problem when using the snapshots. I simply disabled it during the tests. Not ideal, but I think that should be ok.

Reply:
Thanks!